### DISCUSSION TITLE
PDNS & PDNS REcursor for one zone

### DISCUSSION THREAD

**Discussion by shamilgadzhimamaev** [PowerDNS/pdns#13299] [Q&A]
Hello. Can someone provide assistance? I'm using PowerDNS 4.5.3 and PowerDNS Recursor. 
I'm setting up a DNS server for my enterprise local network. We manage the "tui.com" zone. 
Initially, I tried the following approach: I installed the Recursor on port 53 and PDNS on port 5353. 
I configured the Recursor to forward all queries to an external DNS server, except for the "tui.com" zone which should be resolved locally on port 5353. Everything works fine, but since the "tui.com" zone is also managed by an external server (which our access group doesn't have access to), I cannot retrieve any values when performing an nslookup from this server because it gets forwarded to the local server. Ideally, I would like to configure it so that if a record is not found locally, it should be forwarded to the external server. 
What considerations should I take into account? 
Thank you.

**Comment by Habbie** [comment]
> Ideally, I would like to configure it so that if a record is not found locally, it should be forwarded to the external server.

You cannot this on a record (hostname) basis. If you have an external tui.com zone, and you also want internal names that are not visible externally, it's best to make another zone (like 'internal.tui.com')
